packages/accounts-password/password_server.js

```diff
@@ -909,6 +909,9 @@ Accounts.sendVerificationEmail = (userId, email, extraTokenData) => {
   const url = Accounts.urls.verifyEmail(token);
   const options = Accounts.generateOptionsForEmail(realEmail, user, url, 'verifyEmail');
   Email.send(options);
+  if (Meteor.isDevelopment) {
+    console.log(`\nVerification email URL (without MIME): ${url}`);
```

Comment:
`(without MINE)` is not needed.
Maybe use the meteor logging facility?
```js
import { Log } from 'meteor/logging';
```

While at it why not add this to reset password, etc. ?

Reply:
What about using the `logging` facility instead of standard console log?

Reply:
Sorry for the lag. I removed `(without MIME)` from the messatge.
I added the same message to reset password and enrollment emails.

I couldn't find any `logging` facility, can you please point to some example?

I saw that Meteor sometimes use `process.stdout.write` e.g. here for  the `client is refreshing` message: https://github.com/jirikrepl/meteor/blob/e85c69680dad10d7bb1be3a28ada62d2b114212e/tools/runners/run-log.js#L191-L197

Reply:
The Meteor logging is just a suggestion:
https://atmospherejs.com/meteor/logging
It is an internal logging tool, I wrote an article about it:
https://dev.to/storytellercz/meteor-logging-facility-215f